modules/nw-ingress-sharding-route-configuration.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * configuring_ingress_cluster_traffic/configuring-in4ress-cluster-traffic-ingress-controller.adoc
+// * networking/routes/route-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ingress-sharding-route-configuration_{context}"]
+= Creating a route for  IngressController sharding
+
+A route allows you to host your application at a particular URL. The following procedure describes how to create a route for Ingress Controller sharding, using the `hello-openshift` application as an example.
```

Comment:
@Miciah thanks for the info!

> I wonder if it might be confusing to say "at a particular URL" when introducing the spec.subdomain feature since the application actually can be at multiple URLs.

Good point, I'll update that sentence. 

> One of the nifty things about spec.subdomain is that it actually makes it possible for a route to expose an application at multiple URLs: one URL for each ingresscontroller that exposes the route, with the form <subdomain>.<domain> (the route's subdomain joined with the ingresscontroller's domain). That isn't necessarily useful for this example.

This is good context for the user and I think the goal of the example should be to sort of set them up to be able repeat the procedure for when they have multiple Ingress Controllers. I think it will be worthwhile to come back to this example when the "Configuring multiple Ingress Controllers" example is ready/published, to see what tweaks are needed here.